GliaWidgetsTests/SecureConversations/AvailabilityTests.swift

```diff
@@ -33,4 +33,71 @@ final class AvailabilityTests: XCTestCase {
         }
         try XCTAssertEqual(XCTUnwrap(receivedResult), .success(.unavailable(.unauthenticated)))
     }
+
+    func testAvailableQueueStatus() throws {
+        var env = Availability.Environment.failing
+        let queueId = UUID.mock.uuidString
+        env.listQueues = { callback in
+            callback([.mock(id: queueId, status: .open, media: [.messaging])], nil)
+        }
+        env.isAuthenticated = { true }
+        env.queueIds = [UUID.mock.uuidString]
+        let availability = Availability.init(environment: env)
```

Comment:
Probably the `init` is not needed. There's several instances of this throughout the tests after this one.

Reply:
Explicit `init` is only for autocompletion. cb49477b043b9f0cb3e4d08800ee7bb814bc664b